services/app-api/serverless.yml

```diff
@@ -21,15 +21,14 @@ custom:
   tableName: cms-spa-form-${self:custom.stage}-change-requests
   spaIdTableName: cms-spa-form-${self:custom.stage}-spa-ids
   oneMacTableName: onemac-${self:custom.stage}-one
-  iamPath: ${env:IAM_PATH, "/"}
-  emailSource: ${opt:emailSource, env:CMS_SPA_FORM_FROM_EMAIL, "spa-reply@cms.hhs.gov"}
-  userAccessEmailSource: ${opt:userAccessEmailSource, env:CMS_USER_ACCESS_FROM_EMAIL, "OneMACAccess@cms.hhs.gov"}
-  reviewerEmail: ${opt:reviewTeamEmail, env:CMS_SPA_FORM_CMS_EMAIL, "OneMAC@cms.hhs.gov"}
-  reviewerCHIPEmail: ${opt:reviewTeamCHIPEmail, env:CMS_CHIP_FORM_CMS_EMAIL, "OneMAC@cms.hhs.gov"}
-  chipCcEmail: ${opt:chipCcEmail, env:CMS_CHIP_FORM_CC_EMAIL, ""}
-  systemAdminEmail: ${opt:systemAdminEmail, env:CMS_SYSTEM_ADMIN_EMAIL, ""}
-  productionNoEmailDebug: ${opt:productionNoEmailDebug, env:NO_EMAIL_DEBUG, "OneMAC@cms.hhs.gov"}
-  testingEmail: ${opt:testingEmail, env:TESTING_TEAM_EMAIL, ""}
+  iamPath: ${ssm:/configuration/${self:custom.stage}/iam/path, ssm:/configuration/default/iam/path, "/"}
+  iamPermissionsBoundaryPolicy: ${ssm:/configuration/${self:custom.stage}/iam/permissionsBoundaryPolicy, ssm:/configuration/default/iam/permissionsBoundaryPolicy, ""}
```

Comment:
I think it is the right call to remove the opt ability from these variables... not sure we want to allow that kind of variability in a deployment

Also.... I think we have had trouble in the past when the dev ops team updated the ssm:/configuration/default/iam/permissionsBoundaryPolicy and did not update the secret as well... so this should help with those sorts of troubles, too!